b/bash/pkg/profile

```diff
@@ -16,4 +16,13 @@ if [ -d /etc/profile.d ] ; then
     unset script
 fi
 
+if [ -d ~/.bashrc.d ]; then
```

Comment:
This should probably add a "are we running bash"? conditional with `and` logic, to avoid the solus situation of zsh users getting tripped up over it?

Can add a similar .zshrc.d section too for good measure in a separate PR?